javascript-carousel/main.js

```diff
@@ -0,0 +1,76 @@
+var $next = document.querySelector('.next');
+var $previous = document.querySelector('.previous');
+var $currentImage = document.querySelector('.first');
+var $radio1 = document.querySelector('.dot1');
+var $radio2 = document.querySelector('.dot2');
+var $radio3 = document.querySelector('.dot3');
+var $radio4 = document.querySelector('.dot4');
+var $radio5 = document.querySelector('.dot5');
+
+var clickCounter = 0;
+var clickBack = 0;
+
+$next.addEventListener('click', function (event) {
+  clickCounter++;
+  if (clickCounter > 0 && clickCounter <= 1) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else if (clickCounter > 1 && clickCounter <= 2) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickCounter > 2 && clickCounter <= 3) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickCounter > 3 && clickCounter <= 4) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
+  changeImage();
+});
+
+$previous.addEventListener('click', function (event) {
+  clickBack++;
+  if (clickBack > 0 && clickBack <= 1) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else if (clickBack > 1 && clickBack <= 2) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickBack > 2 && clickBack <= 3) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickBack > 3 && clickBack <= 4) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
```

Comment:
Same here, please adjust your conditions to match the format I described above

Reply:
![Screen Shot 2021-02-17 at 6 31 06 PM](https://user-images.githubusercontent.com/74885918/108580142-0a3d6000-72df-11eb-83fd-a0bce1d75fe1.png)